test/e2e/openshift_test.go

```diff
@@ -0,0 +1,68 @@
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jenkinsci/kubernetes-operator/pkg/apis/jenkins/v1alpha2"
+)
+
+const openshfitE2e = "openshift-base-configuration-e2e"
+
+func TestBaseOpenShiftConfiguration(t *testing.T) {
+	t.Parallel()
+	namespace, ctx := setupTest(t)
+	defer showLogsAndCleanup(t, ctx)
+
+	numberOfExecutors := 6
+	numberOfExecutorsEnvName := "NUMBER_OF_EXECUTORS"
+	stringData := make(map[string]string)
+	stringData[numberOfExecutorsEnvName] = fmt.Sprintf("%d", numberOfExecutors)
+
+	// base
+	createUserConfigurationSecret(t, namespace, stringData)
+	createUserConfigurationConfigMap(t, namespace, numberOfExecutorsEnvName, "")
+	sample := JenkinsSample{
+		name:              openshfitE2e,
+		namespace:         namespace,
+		priorityClassName: "",
+		seedJob:           &[]v1alpha2.SeedJob{},
+		groovyScripts:     v1alpha2.GroovyScripts{},
+		casc:              v1alpha2.ConfigurationAsCode{},
+	}
+	jenkins := createJenkinsCRFromSample(t, sample)
+	createDefaultLimitsForContainersInNamespace(t, namespace)
```

Comment:
```suggestion
	createDefaultLimitsForContainersInNamespace(t, namespace)
```
It's not needed in this test.

Reply:
removed